ReactAndroid/src/main/java/com/facebook/react/uimanager/UIManagerModule.java

```diff
@@ -539,6 +551,37 @@ public void sendAccessibilityEvent(int tag, int eventType) {
     mUIImplementation.sendAccessibilityEvent(tag, eventType);
   }
 
+  public void emitUpdateDimensionsEvent() {
```

Comment:
I know you're not adding this, but seems like we should be sharing code with where we create this map when exposing the initial constants.